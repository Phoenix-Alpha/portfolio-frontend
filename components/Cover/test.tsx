import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Cover } from '.'

describe('<Cover />', () => {
  it('should renders correct <Cover />', () => {
    const wrapper = shallow(<Cover src="test.png" />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
