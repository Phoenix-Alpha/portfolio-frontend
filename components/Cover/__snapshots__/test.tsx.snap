// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cover /> should renders correct <Cover /> 1`] = `
<figure
  className="relative mb-8 lg:mb-16"
>
  <Image
    alt="cover"
    className="lg:rounded"
    layout="fill"
    objectFit="cover"
    objectPosition="center"
    src="test.png"
  />
</figure>
`;
